GAMEFLOW.md

```diff
@@ -0,0 +1,1306 @@
+    "type": "cutscene",
+    "music": 0,
+    "sequence": [
+        {"type": "start_cine"},
+        {"type": "set_cam_x", "value": 36668},
+        {"type": "set_cam_z", "value": 63180},
+        {"type": "set_cam_angle", "value": -23312},
+        {"type": "play_synced_audio", "audio_id": 23},
+        {"type": "loop_cine"},
+        {"type": "stop_cine"},
+        {"type": "level_stats", "level_id": 4},
+        {"type": "exit_to_level", "level_id": 5},
+    ],
+    "strings": {},
+},
+```
+</details>
+
+## Item drops
+In the original game, items dropped by enemies were hardcoded such that only
+specific enemies could drop, and the items and quantities that they dropped were
+immutable. This is no longer the case, with the gameflow providing a mechanism
+to allow the _majority_ of enemy types to carry and drop items. Note that this
+also means by default that the original enemies who did drop items will not do
+so unless the gameflow has been configured as such.
+
+Item drops are defined in the `item_drops` section of a level's definition by
+creating objects with the following parameter structure. You can define at most
+one entry per enemy, but that definition can have as many drop items as
+necessary (within the engine's overall item limit).
+
+<details>
+<summary>Show example setup</summary>
+
+```json5
+{
+    "title": "Example Level",
+    "file": "data/example.phd",
+    "type": "normal",
+    "music": 57,
+    "item_drops": [
+        {"enemy_num": 17, "object_ids": [86]},
+        {"enemy_num": 50, "object_ids": [87]},
+        {"enemy_num": 12, "object_ids": [93, 93]},
+        {"enemy_num": 47, "object_ids": [111]},
+    ],
+    "sequence": [
+         {"type": "start_game"},
+         {"type": "loop_game"},
+         {"type": "stop_game"},
+         {"type": "level_stats", "level_id": 1},
+         {"type": "exit_to_level", "level_id": 2},
+    ],
+    "strings": {
+        "key1": "Silver Key",
+        "puzzle2": "Machine Cog",
+    },
+},
+```
+
+This translates as follows.
+- Enemy #17 will drop the magnums
+- Enemy #50 will drop the uzis
+- Enemy #12 will drop two small medipacks
+- Enemy #47 will drop puzzle 2
+</details>
+
+<table>
+  <tr valign="top" align="left">
+    <th>Parameter</th>
+    <th>Type</th>
+    <th>Description</th>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>enemy_num</code>
+    </td>
+    <td>Integer</td>
+    <td>The index of the enemy in the level's item list.</td>
+  </tr>
+  <tr valign="top">
+    <td>
+      <code>object_ids</code>
+    </td>
+    <td>Integer array</td>
+    <td>
+      A list of item <em>types</em> to drop. These items will spawn dynamically
+      and do not need to be added to the level file. Duplicate IDs are permitted
+      in the same array.
+    </td>
+  </tr>
+</table>
+
+You can also toggle `convert_dropped_guns` in
+[global properties](#convert-dropped-guns) so that guns that Lara already has
+are converted to ammo when they are dropped by enemies.
```

Comment:
`You can also toggle `convert_dropped_guns` in
[global properties](#convert-dropped-guns) so that dropped guns Lara already has
are converted to ammo when they are dropped by enemies.`

Slight reword. What do you think? The `that` sandwich was a little confusing to read.

Reply:
Yes, agreed, it wasn't too clear. I've made it super-explicit now - above sounded good, but the double "dropped" was making me hesitate a little 😄